tests/unit/test_main.py

```diff
@@ -111,10 +112,12 @@ def test_update_colors(self, req_post_mock):
         """Test method update_colors."""
         switch1 = Mock()
         switch1.dpid = '00:00:00:00:00:00:00:01'
+        switch1.status = EntityStatus.UP
```

Comment:
This unit test looks good, but let's also at least unit test that if a switch status isn't `UP` then no requests of that switch are expected to be sent. Can you cover this as well?

Reply:
Utilized the same test for `DOWN` switches case.